resources/user-guides/getting-started-with-adobe-experience-platform/README.md

```diff
@@ -165,16 +165,16 @@ AEP Assurance (also known as Project Griffon) is a product from Adobe that lets
     If you are using a real device, you can also use the `Scan QR Code` functionality. You can then skip to step 6.
     {% endhint %}
 
-   ​![](https://firebasestorage.googleapis.com/v0/b/gitbook-28427.appspot.com/o/assets%2F-Lf1Mc1caFdNCK_mBwhe%2Fsync%2F562452794e4fd8a786f42826ad2ebf972ffa5efd.png?generation=1602553739064019&alt=media)​
+   ![](https://firebasestorage.googleapis.com/v0/b/gitbook-28427.appspot.com/o/assets%2F-Lf1Mc1caFdNCK_mBwhe%2Fsync%2F562452794e4fd8a786f42826ad2ebf972ffa5efd.png?generation=1602553739064019&alt=media)​
```

Comment:
It may just be an artifact of GH's rich text viewer, but the images in this "using AEP Assurance" section appear really big.  If they are indeed really big after publishing, you may need to resize them.